### DISCUSSION TITLE
Pressure freeze potential fix

### DISCUSSION THREAD

**Discussion by banoz** [Zer0-bit/gaggiuino#224] [General]
https://github.com/Zer0-bit/gaggiuino/blob/d123dc83bdc28c72bfd3014afba3e6fcc453a2f5/src/gaggiuino.ino#L363

Maybe replace the line with

```
      int16_t adcValue = ADS.getValue();
      if (adcValue == 0){
        ads1115Init();        
        adcValue = ADS.getValue();
      }
      return adcValue / 1706.6f - 1.49f;
```

**Comment by Zer0-bit** [comment]
Not sure about this since it never froze for ne nor does it for anyone on stm32 apparently
